bot/role_management/role_management.py

```diff
@@ -58,6 +58,7 @@ async def toggle_module(self, ctx: commands.Context, *, str_modules: str):
         for module in modules:
             module_upper = module.upper()
             try:
+                # TODO: Check if role is whitelisted.
```

Comment:
I assume this is in the other PR I approved?

Reply:
Yep.